inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -217,11 +217,22 @@ def is_experimental_paper(obj, eng):
 @with_debug_logging
 def is_arxiv_paper(obj, *args, **kwargs):
     """Check if the record is from arXiv."""
-    arxiv_id = get_arxiv_id(obj.data)
-    categories = get_value(obj.data, 'arxiv_eprints.categories')
 
-    if arxiv_id or categories:
-        return True
+    acquisition_source = obj.data.get('acquisition_source')
+
+    if acquisition_source:
+        method = acquisition_source.get('method')
+
+        if method == 'submitter':
+            arxiv_id = get_arxiv_id(obj.data)
+            categories = get_value(obj.data, 'arxiv_eprints.categories')
+            if arxiv_id or categories:
+                return True
```

Comment:
this can be simplified thanks to the strict requirements of the schema: presence of `arxiv_eprints` is sufficient to know that there is both an `arxiv_id` and `categories`.
```python
if method == 'submitter':
    return 'arxiv_eprints' in obj.data
```